examples/tune_pytorch/utils.py

```diff
@@ -46,67 +40,41 @@ def train(model, optimizer, train_loader, epoch_size=512, smoke_test=True):
         optimizer.step()
 
 
-def test(model, data_loader, test_size=256, smoke_test=True):
+def test(model, data_loader, epoch_size=256, smoke_test=True):
     device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
     model.eval()
+
     correct = 0
     total = 0
+
     with torch.no_grad():
         for batch_idx, (data, target) in enumerate(data_loader):
-            if smoke_test and batch_idx * len(data) > test_size:
+            # exit early only on a small subset of data
+            if smoke_test and batch_idx * len(data) > epoch_size:
                 break
             data, target = data.to(device), target.to(device)
             outputs = model(data)
             _, predicted = torch.max(outputs.data, 1)
             total += target.size(0)
             correct += (predicted == target).sum().item()
-
+    
     return correct / total
 
 
-def download_mnist():
-    subprocess.run(
-        ["wget", "www.di.ens.fr/~lelarge/MNIST.tar.gz"],
-        stdout=subprocess.DEVNULL,
-        stderr=subprocess.STDOUT,
-    )
-    subprocess.run(
-        ["tar", "-zxvf", "MNIST.tar.gz"],
-        stdout=subprocess.DEVNULL,
-        stderr=subprocess.STDOUT,
-    )
-
-
 def train_mnist(
-    config, epoch_size, test_size, dataset="mnist", smoke_test=True, n_epochs=10
+    config, epoch_size, test_size, smoke_test=True, n_epochs=10
 ):
-    # Data Setup
-    if dataset == "mnist":
-        try:
-            download_mnist()
-            mnist_transforms = transforms.Compose(
-                [transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))]
-            )
-            train_loader = DataLoader(
-                datasets.MNIST(
-                    "./", train=True, download=True, transform=mnist_transforms
-                ),
-                batch_size=64,
-                shuffle=True,
-            )
-            test_loader = DataLoader(
-                datasets.MNIST(
-                    "./", train=False, download=True, transform=mnist_transforms
-                ),
-                batch_size=64,
-                shuffle=True,
-            )
-        except (ValueError, RuntimeError, EOFError) as e:
-            print("Issue with MNIST dataset path.")
-            session.report({"mean_accuracy": 0.0})
-            return
-    else:
-        raise ValueError("Only MNIST dataset is currently supported by this interface.")
+    mnist_transforms = transforms.Compose([transforms.ToTensor(),transforms.Normalize((0.1307,), (0.3081,))])
+    train_loader = DataLoader(
+        datasets.MNIST("./", train=True, download=True, transform=mnist_transforms),
+        batch_size=64,
+        shuffle=True,
+    )
+    test_loader = DataLoader(
+        datasets.MNIST("./", train=False, download=True, transform=mnist_transforms),
+        batch_size=64,
```

Comment:
batch_size is statically set here. This shouldn't be the case given there that we set a `test_size` somewhere.